portscanner.py

```diff
@@ -51,6 +53,21 @@ def tcp_threader():
         port = q.get()
         tcp_scan(port)
         q.task_done()
+        
+def udp_scan(port):
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    try:
+        s.sendto('invalid request', (host, port))
+        with print_lock:
+            print('%-15s%-15s%-15s' % (str(port) +'/udp', 'open', getPortService(port, 'udp')))
```

Comment:
So should the port status be `open` or something else, like `unclosed` or `possibly-open`?

Reply:
right, how does `open | filter` sound?

Reply:
what is the `filter` for?

Reply:
for packets rejected by firewall(filtered)

Reply:
Ok, make it `filtered` instead of `filter`

Reply:
will do that